.travis.yml

```diff
@@ -1,15 +1,28 @@
 language: crystal
+
+crystal:
+  - latest
+  - nightly
+
+jobs:
+  allow_failures:
+    - crystal: nightly
+
+install:
+  - shards install
+
 script:
   - make test
   - crystal tool format --check
   - sed -i -e 's:<.*>::g' README.md
   - crystal docs
+
 deploy:
   provider: pages
-  github_token: $GITHUB_TOKEN
```

Comment:
please rever the unrelated changes. Moreover, `github_token` is an alias and imo just works here.

Reply:
Will do. Being an alias, adds to the Travis CI validation notes, so I reckoned it's better to use the actual key name instead.